docs/modules/ROOT/pages/vshn-managed/redis/maintenance.adoc

```diff
@@ -2,17 +2,17 @@
 
 By default, there will be a security maintenance on Tuesday night (Central Europe Time).
 
-NOTE: For more information about maintenance procedures, please visit https://products.docs.vshn.ch/products/appcat/redis.html#_recurring_maintenance[the product description].
+NOTE: For more information about maintenance procedures, please visit https://products.docs.vshn.ch/products/appcat/mariadb.html#_recurring_maintenance[the product description].
```

Comment:
file is called redis/maintenance but contains mariadb references

Reply:
Fixed